ingestion/src/metadata/ingestion/api/parser.py

```diff
@@ -329,7 +329,10 @@ def parse_workflow_config_gracefully(
         workflow_config = OpenMetadataWorkflowConfig.parse_obj(config_dict)
         return workflow_config
 
-    except ValidationError:
+    except ValidationError as e:
+        if e.model.__name__ == OpenMetadataWorkflowConfig.__name__:
+            logger.error(e)
```

Comment:
Could we have more information instead of just logging the exception? 
e.g.
```suggestion
            logger.error(f"Failed when parsing the workflow config gracefully: {e}")
```